website/templates/project/register_draft.mako

```diff
@@ -0,0 +1,60 @@
+<%inherit file="project/project_base.mako"/>
+
+<%def name="title()">Register ${node['title']}</%def>
+
+<%def name="content()">
+<div class="col-md-6 col-md-offset-3" id="draftRegistrationScope">        
+      <div id="embargo-addon" data-bind="with: $root.embargoAddon">
+        <hr />
+        
+        <p class="help-block">${language.REGISTRATION_EMBARGO_INFO}</p>
```

Comment:
If at all possible, avoid inserting HTML into language.py and just have separate chunks of text.

If you absolutely must keep the HTML in, you will need to explicitly exempt this string with the mako  `n` filter. Otherwise the `<p>` tags will be rendered as written, instead of treated as code.